foxglove_bridge_base/include/foxglove_bridge/server_interface.hpp

```diff
@@ -13,6 +14,8 @@ namespace foxglove {
 
 constexpr size_t DEFAULT_SEND_BUFFER_LIMIT_BYTES = 10000000UL;  // 10 MB
 
+typedef std::unordered_map<std::string, std::unordered_set<std::string>> MapOfSets;
```

Comment:
nit: I personally prefer `using` everywhere since it feels more readable
```suggestion
using MapOfSets = std::unordered_map<std::string, std::unordered_set<std::string>>;
```